test/bin/scenario.sh

```diff
@@ -1189,6 +1204,12 @@ action_run() {
 
     check_dependencies
 
+    if [ $# -eq 0 ]; then
+        RUN_HOST_OVERRIDE="host1"
```

Comment:
We should not be setting artificial values in this variable. 
Let's keep it empty when not specified?

Reply:
Done.